packages/core/src/external/fhir/normalization/resources/coverage.ts

```diff
@@ -0,0 +1,22 @@
+import { Coverage } from "@medplum/fhirtypes";
+import { isValidUuid } from "../../../../util/uuid-v7";
+
+export function normalizeCoverages(coverages: Coverage[]): Coverage[] {
+  return coverages.map(coverage => {
+    const identifiers = coverage.identifier;
+    if (!identifiers) return coverage;
+
+    const validIdentifiers = identifiers.filter(identifier => {
+      const value = identifier.value?.trim();
+      if (!value) return false;
+
+      if (value.includes("urn:uuid")) return false;
```

Comment:
Why don't we want Coverage with UUID?

Reply:
We're going to be storing identifiers that are useful for us and our users - those, that might contain the insurance policy ID. UUIDs are not useful, but very commonly end up in there.